doc/usecases/polyculture/suggest_groups.md

```diff
@@ -0,0 +1,31 @@
+# Use Case: Group suggestions
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user is provided with suggestions on groups of plants to use.
```

Comment:
identical to other use case?

Reply:
I see what you mean.
I deleted the suggest groups use case and described it as an alternative scenario in the suggest plants use case.